docs/email/postfix/email-with-postfix-dovecot-and-mysql.md

```diff
@@ -310,29 +311,41 @@ Next, set up Postfix so the server can accept incoming messages for the domains.
         query = SELECT destination FROM virtual_aliases WHERE source='%s'
         ~~~
 
-6.  Save the changes you've made to the `/etc/postfix/mysql-virtual-alias-maps.cf` file, and restart Postfix with the following command.
+6.  Create the `/etc/postfix/mysql-virtual-email2email.cf` file and enter the following values. Again, make sure you use the mailuser's password, and make any other changes as necessary.
+
+      {: .file }
+      /etc/postfix/mysql-virtual-email2email.cf
+      : ~~~
+        user = mailuser
+        password = mailuserpass
+        hosts = 127.0.0.1
+        dbname = mailserver
+        query = SELECT email FROM virtual_users WHERE email='%s'
+        ~~~
+
+7.  Save the changes you've made to the `/etc/postfix/mysql-virtual-email2email.cf` file, and restart Postfix with the following command.
 
         sudo service postfix restart
 
-7.  Enter the following command to ensure that Postfix can find the first domain. Be sure to replace `example.com` with the first virtual domain. The command should return `1` if it is successful.
+8.  Enter the following command to ensure that Postfix can find the first domain. Be sure to replace `example.com` with the first virtual domain. The command should return `1` if it is successful.
```

Comment:
I'm retesting this guide using your version. I'm not sure if this stems from your changes or it was a preexisting issue, but when I test other virtual domains in this table, this command still returns `1`. Is 1 supposed to be the row in the table where that domain is listed, or is it just a binary answer?